packaging_automation/tests/test_package_cloud_statistics_collector.py

```diff
@@ -20,15 +24,22 @@
 def test_fetch_and_save_package_cloud_stats():
     db = create_engine(db_connection_string(db_params=db_parameters, is_test=True))
     db.execute(text(f'DROP TABLE IF EXISTS {PackageCloudDownloadStats.__tablename__}'))
-    pack_count = package_count(ORGANIZATION, REPO, PACKAGE_CLOUD_API_TOKEN)
     session = db_session(db_params=db_parameters, is_test=True)
     page_record_count = 3
     parallel_count = 3
+
+    result = stat_get_request(
+        package_list_with_pagination_request_address(PACKAGE_CLOUD_API_TOKEN, 1, ORGANIZATION, REPO, 100),
+        RequestType.package_cloud_list_package, session)
+    package_info_list = json.loads(result.content)
+    package_list = list(filter(
+        lambda p: not is_ignored_package(p["name"]), package_info_list))
+
     for index in range(0, parallel_count):
         fetch_and_save_package_cloud_stats(package_cloud_api_token=PACKAGE_CLOUD_API_TOKEN, organization=ORGANIZATION,
                                            repo_name=REPO, db_params=db_parameters, parallel_count=parallel_count,
                                            parallel_exec_index=index, page_record_count=page_record_count,
                                            is_test=True, save_records_with_download_count_zero=True)
 
     records = session.query(PackageCloudDownloadStats).all()
-    assert len(records) == pack_count * PACKAGE_SAVED_HISTORIC_RECORD_COUNT
+    assert len(records) == len(list(package_list)) * PACKAGE_SAVED_HISTORIC_RECORD_COUNT
```

Comment:
This was already a list I don't think you need to cast it again

Reply:
Yes you're right fixed